diamm/serializers/website/stats.py

```diff
@@ -0,0 +1,11 @@
+import serpy
+
+from diamm.serializers.serializers import ContextDictSerializer
+
+class StatsSerializer(ContextDictSerializer):
+    source = serpy.IntField(label='sources')
```

Comment:
I was using the label field as a simple way to make sure the titles were pluralized. Should I use a filter in the template instead? That approach would be less fun when we encounter titles that don't just append an 's' e.g. person -> people